src/commands/jobs/util.ts

```diff
@@ -57,4 +57,83 @@ const detailOutputFormat = {
 	]
 };
 
-export { listOutputFormat, detailOutputFormat };
+const historyOutputFormat = {
+	table: [
+		{
+			header: 'JobID',
+			property: 'job_id'
+		},
+		{
+			header: 'Progress',
+			property: 'progress'
+		},
+		{
+			header: 'Phase',
+			property: 'phase'
+		},
+		{
+			header: 'StartTime',
+			property: 'start_time',
+			transform: value => new Date(value)
+		},
+		{
+			header: 'EndTime',
+			property: 'end_time',
+			transform: value => new Date(value)
+		},
+		{	
+			header: 'Status',
+			property: 'status'
+		},
+		{	
+			header: 'NodeAddress',
+			property: 'node_address'
+		}
+	],
+	json: [
+		'stats',
+		'duration'
+	]
+};
+
+const auditOutputFormat = {
+	table: [
+		{
+			header: 'JobID',
+			property: 'job_id'
+		},
+		{
+			header: 'ExecutionID',
+			property: 'execution_id'
+		},
+		{
+			header: 'FromSource',
+			property: 'from_source'
+		},
+		{
+			header: 'ToDestination',
+			property: 'to_destination',
+			transform: value => new Date(value)
+		},
+		{
+			header: 'RecordedOn',
+			property: 'recorded_on',
+			transform: value => new Date(value)
+		},
+		{	
+			header: 'Level',
+			property: 'level'
+		},
+		{	
+			header: 'Event',
+			property: 'event'
+		}
```

Comment:
Should `message` be included?

Reply:
Do you mean `reason` ?  If so, I will add.